### ISSUE TITLE
Build fails with notify 5.0.0-pre.11

### ISSUE THREAD

**Issue by micwoj92** [golddranks/sharp_pencil#2]
Currently Cargo.toml specifies notify 5.0.0-pre.3, but due to how cargo resolves dependencies it pulls latest 5.0.0-pre.15 which has some incompatible changes. Latest version that works is 5.0.0-pre.10.

Notify change which causes this: https://github.com/notify-rs/notify/pull/336

This is error when compiling with notify 5.0.0-pre.11
```
error[E0599]: no function or associated item named `new_immediate` found for trait object `dyn Watcher` in the current scope
  --> src/serving.rs:52:28
   |
52 |         watcher = Watcher::new_immediate(watch_files(registry)).unwrap();
   |                            ^^^^^^^^^^^^^ function or associated item not found in `dyn Watcher`

error[E0308]: mismatched types
  --> src/serving.rs:54:23
   |
54 |         watcher.watch(&template_dir, RecursiveMode::Recursive).unwrap();
   |                       ^^^^^^^^^^^^^ expected struct `std::path::Path`, found struct `std::string::String`
   |
   = note: expected reference `&std::path::Path`
              found reference `&std::string::String`

```

**Comment by golddranks** [comment]
I guess that we could specify an exact dependency to 5.0.0-pre.3 or 5.0.0-pre.10, and maybe fix the problem once the final 5.0.0 is released...?

**Comment by golddranks** [comment]
Sorry for the cowboy-style update, but I went and did exactly that. 0.6.1 is out now. Let me know if something is still broken.

**Comment by micwoj92** [comment]
> I guess that we could specify an exact dependency to 5.0.0-pre.3 or 5.0.0-pre.10, and maybe fix the problem once the final 5.0.0 is released...?

I agree that it's better to wait for final 5.0.0
